[MB-14452] Add W-2 disclaimer on About My PPM page

bellex3 opened the pull request:
## [MB-14452](https://dp3.atlassian.net/jira/software/c/projects/MB/boards/26?modal=detail&selectedIssue=MB-14452) for this change

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

<!-- Fill out the next section as you see fit, these are just suggestions. -->

1. Go to milmovelocal:3000.
2. Click Local sign-in & log in as an existing user called "readyToFinish@ppm.approved".
3. After logging in, scroll down to section 5 "Manage your PPM" and click on the blue button on the right that says "Upload PPM Documents".
4. The new W-2 disclaimer text, new headline, and text styling changes will be there. 
## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

<!-- Fill out the next sections as you see fit, these are just suggestions. -->

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/main/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

### Database

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

## Screenshots
<img width="846" alt="Screen Shot 2022-11-14 at 8 40 35 AM" src="https://user-images.githubusercontent.com/110994428/201718047-b7621dce-765c-4fbc-8955-2b2b263c031c.png">

robot-mymove commented:
<!--
  0 failure: 
  1 warning:  This PR does not ...
  1 messages
  
  DangerID: danger-id-Danger;
-->


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>This PR does not include changes to unit tests, even though it affects app code.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:link: <a href="https://dp3.atlassian.net/browse/MB-14452">MB-14452</a></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f92f26a2afa00602dbbdf65550cb52f80c0864bc
</p>

duncan-truss commented on src/pages/MyMove/PPM/Closeout/About/About.jsx:
```diff
@@ -76,7 +76,10 @@ const About = () => {
             <div className={closingPageStyles['closing-section']}>
-              <p>Finish moving this PPM before you start documenting it.</p>
+              <p>
+                <b>Finish moving this PPM before you start documenting it.</b>
```
I think we usually don't use the `<b>` tag because it introduces a presentational element in with the HTML instead of through separating it into CSS.

I would suggest we style this paragraph using the `font-weight` property in CSS.  Other choices could be the `<strong>` or `<em>` elements. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

bellex3 replied:
@duncan-truss  Done, deleted the <b> tag and added the font-weight in the CSS

duncan-truss commented on src/pages/MyMove/PPM/Closeout/Closeout.module.scss:
```diff
@@ -6,10 +6,27 @@
   }
 
-  p:last-of-type {
+  p:nth-of-type(1){
     @include u-margin-bottom(0);
+    @include u-font-weight(bold);
+  }
+
+  h2 {
+    @include u-margin-top(4);
+    @include u-margin-bottom(4);
+  }
+
+  p:nth-of-type(2), p:nth-of-type(3) {
+    @include u-margin-top(0);
+    @include u-margin-bottom(0);
+  }
+
+  p:last-of-type {
+    @include u-margin-top(4);
+    @include u-margin-bottom(4);
```
The `.closing-section` class is shared also with the WeightTickets and ProGear page components.  With this kind of CSS selector I think you are seeing some unintended side effects on those pages now.  It might be a better route to assign specific classnames to the paragraphs that you want to style so it's not a surprise when other rules get added in.

<img width="722" alt="Screen Shot 2022-11-15 at 4 36 42 PM" src="https://user-images.githubusercontent.com/52669884/202030903-565e859b-76a7-4497-aa39-1d97827bfdf2.png">
<img width="737" alt="Screen Shot 2022-11-15 at 4 35 55 PM" src="https://user-images.githubusercontent.com/52669884/202030905-b7d63f89-da60-45ae-8b63-c0ed061bc0b6.png">

bellex3 replied:
Ooh, did not realize the styling changes affected the other pages. Thank you. I have updated the css to add a specific class for the about ppm page. Also checked the pro-gear and weight ticket pages, and they are good now.

duncan-truss reviewed (approved):
Code changes look good, I didn't vet the designs too closely since you're getting a design review.

ronaktruss commented:
From a code perspective this looks good to me as well! I saw some style mismatches for mobile. Specifically the h2 tags still seem to have top and bottom margins of 32px where the Figma mocks show 24px margins.
Eg.
![image](https://user-images.githubusercontent.com/84742904/202553126-31dceadf-f487-44db-a37b-af1140c0b020.png)


Unrelated to your changes, I also noticed that our spacing for our h1 on our pages is not consistent with the other pages in our app. It's being computed as 26.834px (0.67em)

bellex3 commented:
> From a code perspective this looks good to me as well! I saw some style mismatches for mobile. Specifically the h2 tags still seem to have top and bottom margins of 32px where the Figma mocks show 24px margins. Eg. ![image](https://user-images.githubusercontent.com/84742904/202553126-31dceadf-f487-44db-a37b-af1140c0b020.png)
> 
> Unrelated to your changes, I also noticed that our spacing for our h1 on our pages is not consistent with the other pages in our app. It's being computed as 26.834px (0.67em)

@ronaktruss Thanks! i fixed the css for the mobile issue

bellex3 commented:
> @bellex3 I appreciate your patience while the deploy bot issues were being sorted out!
> 
> I didn't notice anything other than what @ronaktruss already mentioned about the mobile h2 tag margins.

@rileyorr no worries & thanks for checking it out. I have fixed the mobile styling issue

ronaktruss commented on src/pages/MyMove/PPM/Closeout/Closeout.module.scss:
```diff
@@ -13,3 +13,34 @@
+    @include u-margin-top(3);
+    @include u-margin-bottom(3);
+
+    @include at-media('desktop') {
+      @include u-margin-top(4);
+      @include u-margin-bottom(4);
+    }
```
@rileyorr Do you know when we should be making the transition to the 24px margins? 

In the past I've always assumed it was at the tablet boundary (640 px), but I don't think I've ever verified this assumption.

bellex3 replied:
@ronaktruss based on Riley's comment, I have changed the desktop breakpoint to tablet

rileyorr commented:
> @rileyorr Do you know at what screen size we should be making the transition to the 24px margins?
> 
> In the past I've always assumed it was at the tablet boundary (640 px), but I don't think I've ever verified this assumption.

@ronaktruss the tablet boundary (640 px) looks correct. I just checked staging and it looks like 640 px is the transition point other parts of the app use (I took a look at the 'Orders' and 'Shipments' sections). 

> @rileyorr no worries & thanks for checking it out. I have fixed the mobile styling issue

@bellex3 thanks for letting me know - I'll check it out this morning (currently waiting on the deploy bot).

rileyorr commented:
@bellex3 the margins on mobile look great - thanks for updating those! However, it looks like the desktop version also has 24px margins instead of 32px. I added screenshots below for reference.

**Mobile:**

<img width="313" alt="Screenshot 2022-11-21 at 10 50 38 AM" src="https://user-images.githubusercontent.com/46329147/203099161-483fe655-8add-473f-a79c-845e76de15d9.png">

**Desktop:**
<img width="529" alt="Screenshot 2022-11-21 at 10 50 55 AM" src="https://user-images.githubusercontent.com/46329147/203099173-8f1801f7-224a-46d8-a1c3-8c54d3bce2c8.png">

bellex3 commented:
> @bellex3 the margins on mobile look great - thanks for updating those! However, it looks like the desktop version also has 24px margins instead of 32px. I added screenshots below for reference.
> 
> **Mobile:**
> 
> <img alt="Screenshot 2022-11-21 at 10 50 38 AM" width="313" src="https://user-images.githubusercontent.com/46329147/203099161-483fe655-8add-473f-a79c-845e76de15d9.png">
> 
> **Desktop:** <img alt="Screenshot 2022-11-21 at 10 50 55 AM" width="529" src="https://user-images.githubusercontent.com/46329147/203099173-8f1801f7-224a-46d8-a1c3-8c54d3bce2c8.png">

@rileyorr Fixed! I changed the desktop breakpoint to tablet

rileyorr reviewed (approved):
@bellex3 thanks for updating the margin - LGTM! Also reviewed the Happo diffs
